### DISCUSSION TITLE
Add support for templates in reviewers/assignees

### DISCUSSION THREAD

**Discussion by stephane-julien** [renovatebot/renovate#33256] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Azure DevOps Server 2022.1

### Please tell us more about your question or problem

I'm running renovate latest (actually 39.82.3) with Azure DevOps Server with a [single pipeline for multiple projects](https://docs.renovatebot.com/modules/platform/azure/#using-a-single-pipeline-to-update-multiple-repositories). I'm trying to add [reviewers](https://docs.renovatebot.com/configuration-options/#reviewers) and need to specify the exact team name as seen in #17728.

With an hardcoded team name "_ProjectName_ Team", it works well. The problem is that the _ProjectName_ change in every repository lookep up. I searched all the teams with [Azure DevOps API](https://learn.microsoft.com/en-us/rest/api/azure/devops/core/teams/get-all-teams?view=azure-devops-server-rest-7.1&tabs=HTTP) and found a Team for every Azure DevOps project, but it is prefixed with _ProjectName_. Example : 
[AzureDevOpsTeams.json](https://github.com/user-attachments/files/18231741/AzureDevOpsTeams.json)

The _ProjectName_ can probably extracted from the repository field according to these logs :
`repository=Project1/Project1GitRepository`
I think this project name can be extracted from the repository with the following template value :
`"reviewers": ["{{ lookup (split repository '/') 0 }} Team"],`

Unfortunately, the reviewers field is not passed through for templating and I can find the following logs
`INFO: {{ lookup (split repository '/') 0 }} Team is neither an Azure DevOps Team nor a user associated with a Team (repository=ACJU.Docker/ACJU.Docker.BaseImage.Dotnet.HelloWorld, baseBranch=master, branch=renovate/master-baseBranchaspnet-dotnet-base-img)`

Am I right with my analysis ? If yes, is it possible to add support of templates for reviewers field ?

**Comment by rarkins** [comment]
Sure, PR welcome
